docs/Installation.md

```diff
@@ -468,7 +450,7 @@ Once the number reaches 100% a JSON object is printed and zmtest terminates.
 If you upgrade your Zonemaster installation with a newer version of
 Zonemaster-Backend and keep the database, then you might have to upgrade the
 database to use it with the new version of Zonemaster-Backend. Please see the
-[upgrade][README.md-upgrade] information.
+[Upgrade document].

```

Comment:
I think the section "Upgrade Zonemaster database" should be moved to the top just before the first installation section, and it should be renamed to "Upgrade of Zonemaster backend". Here it could be noted that you should remove the database first if you do not want to upgrade. Etc.

Reply:
Any comment on this suggestion?